templates/ddd-service-native-db/.env.example

```
@@ -1,33 +1,53 @@
 
-# SQLite (when DB_BACKEND=sqlite)
+# --- DB_BACKEND=sqlite only -------------------------------------------------------------
 DB_PATH=./data/app.db
 
-# All other backends — set DB_DSN to skip individual fields
+# --- every backend EXCEPT sqlite ---------------------------------------------------------
+# Set DB_DSN to skip the individual fields.
 DB_USER=app_user
 DB_PASSWORD=changeme
 DB_HOST=localhost
 DB_PORT=5432        # varies by backend: postgresql=5432, mysql/mariadb=3306, mssql=1433, oracle=1521
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PORT key should go before the DB_USER key

(UnorderedKey)